docs/manual/manual_stow.rst

```diff
@@ -21,10 +21,8 @@ To see if your module and firmware style supports this feature, please see our :
 Stow Process Overview
 ======================
 When a module moves to its stow position, regardles of the reason for the stow, it follows a series of steps to stow properly. First, it begins either accelerating or decelerating
-in order to reach its stow position with its user-configrued :ref:`stow acceleration <stow_movement_parameters>`. So if the module was still before, it will begin moving, but if it was 
-spinning rapdily, it will keep spinning and gradually slow down. Once the module has come to a stop at the stow position, it will then choose whether it should try to 
-actively :ref:`hold that position <stow_holding_position>` or coast itself, meaning that the module will not be driving itself and can spin freely. If the module is :ref:`interrupted <interrupting_stow>` at any time during 
-the stow process, it will stop trying to stow.
+in order to reach its stow position with its user-configured :ref:`stow acceleration <stow_movement_parameters>`. So if the module was still before, it will begin moving, but if it was 
+spinning rapdily, it will keep spinning and gradually slow down. Once the module has come to a stop at the stow position, it will then perform its :ref:`ending hold behavior <stow_holding_position>`, which has multiple options.

```

Comment:
maybe, "it will then perform its ending hold behavior, which has multiple options discussed below" and put the link on the discussed below?